src/test/java/com/google/cloud/spanner/pgadapter/JdbcSimpleModeMockServerTest.java

```diff
@@ -259,6 +508,366 @@ public void testDdl() throws SQLException {
             .collect(Collectors.toList());
     assertEquals(1, updateDatabaseDdlRequests.size());
     assertEquals(1, updateDatabaseDdlRequests.get(0).getStatementsCount());
-    assertEquals(sql, updateDatabaseDdlRequests.get(0).getStatements(0));
+    assertEquals(
+        "CREATE TABLE foo (id bigint primary key)",
+        updateDatabaseDdlRequests.get(0).getStatements(0));
+
+    // Verify that the DML statements were batched together by PgAdapter.
+    List<ExecuteBatchDmlRequest> requests =
+        mockSpanner.getRequestsOfType(ExecuteBatchDmlRequest.class);
+    assertEquals(1, requests.size());
+    ExecuteBatchDmlRequest request = requests.get(0);
+    assertEquals(2, request.getStatementsCount());
+    assertEquals(INSERT_STATEMENT.getSql(), request.getStatements(0).getSql());
+    assertEquals(UPDATE_STATEMENT.getSql(), request.getStatements(1).getSql());
+  }
```

Comment:
nit: Add an assertion that we also received one `CommitRequest`.

Reply:
Done.